### DISCUSSION TITLE
Info in Status Page

### DISCUSSION THREAD

**Discussion by ArneTR** [green-coding-solutions/green-metrics-tool#622] [Ideas]
With PR https://github.com/green-coding-berlin/green-metrics-tool/pull/599 we have implemented a more detailed status page that not only shows the job-queue but also shows information regarding the expected waiting time and when the machine was last een online.

The current version is online on: https://metrics.green-coding.berlin/status.html (Please disable cache if you cannot see the machines tabletc)

I can imagine some more information on this page to be useful:
- Which CPU does the machine have
- How much RAM does the machine have
- Does the machine have a dedicated GPU ?
- How much Cores / Threads does the machine have
- What version of the GMT is the machine running

@ribalba What more information would you like to see?

@davidkopp  What more information would you like to see?

@dan-mm What more information would you like to see?

**Comment by davidkopp** [comment]
I like that!

Some comments about the current view:
- what does "status" actually mean? There are two machines with status `job_no`, one with `job_start` and the others don't have a status. I think at least for all available machines there should be a status shown, e.g. like waiting or busy.
- the description at the top of the page is outdated.
- the description at the top states that there are two processing schemes, random or FIFO. Maybe this configuration could also be something to show in the table.

I, as a contributor, would find it useful to see the GMT version that is running on the machines.

At the moment I'm only using one machine, so I don't need to see all the hardware data of the machines on the status dashboard. The information given in the [documentation](https://docs.green-coding.berlin/docs/measuring/measurement-cluster/) is sufficient for me.

**Comment by ribalba** [comment]
As @davidkopp already mentioned it is unclear what Status actually is or what does something like "job_no" mean. For us this is clear but for someone external it might be confusing. Otherwise I really like it

**Comment by ArneTR** [comment]
I have incorporated all the additons here: https://github.com/green-coding-berlin/green-metrics-tool/pull/641

Thanks for your feedback, y'all.
